src/main/java/com/spring/onlinebookstore/model/Book.java

```diff
@@ -26,6 +30,10 @@ public class Book {
     private String isbn;
     @Column(nullable = false)
     private BigDecimal price;
+
+    @Column(columnDefinition = "tinyint(1) default 0", nullable = false)
+    private boolean isDeleted = false;
```

Comment:
`@Column(nullable = false)
@Type(type = "org.hibernate.type.NumericBooleanType")
private boolean isDeleted = false;
`
try to use this way